src/services/ExplorerService/fetcher.ts

```diff
@@ -934,3 +934,23 @@ export type UDTQueryResult = {
   typeHash: string
   iconFile: string | null
 }
+
+type SubmitTokenInfoParams = {
+  typeHash: string
+  symbol: string
+  email: string
+  operator_website: string
+
+  full_name?: string
+  decimal?: number
+  total_amount?: number
+  description?: string
+  icon_file?: string
+  display_name?: string
+  uan?: string
+  token?: string
```

Comment:
Does the API `PUT /udts` define params that mix upper and lower case? Is the first parameter `typeHash` not the same with the `typeHash` in `SubmitTokenInfoParams`

Reply:
change to lower case when calling API